Plugin/Source/GUI/Assets/gui.xml

```diff
@@ -1,246 +1,260 @@
+            <TextButton margin="0" padding="2" text="3.75" button-color="00000000" background-color="00000000"
+                        onClick="set_speed_3.75" lookAndFeel="SpeedButtonLNF" button-on-color="00000000"
+                        name="3.75 ips" tooltip="Snaps the tape speed to 3.75 inches per second."/>
+            <TextButton text="7.5" margin="0" padding="2" button-color="00000000" background-color="00000000"
+                        onClick="set_speed_7.50" lookAndFeel="SpeedButtonLNF" button-on-color="00000000"
+                        name="7.5 ips" tooltip="Snaps the tape speed to 7.5 inches per second."/>
+            <TextButton margin="0" padding="2" text="15" button-color="00000000" button-on-color="00000000"
+                        background-color="00000000" onClick="set_speed_15.00" lookAndFeel="SpeedButtonLNF"
+                        name="15 ips" tooltip="Snaps the tape speed to 15 inches per second."/>
+            <TextButton margin="0" padding="2" background-color="00000000" onClick="set_speed_30.00"
+                        lookAndFeel="SpeedButtonLNF" text="30" button-color="00000000"
+                        button-on-color="00000000" name="30 ips" tooltip="Snaps the tape speed to 30 inches per second."/>
+          </View>
+          <View flex-grow="0.01" background-color="00000000"/>
+          <PowerButton margin="0" padding="0" background-color="00000000" max-height="25"
+                       min-height="20" button-color="ff595c6b" button-on-color="FFEAA92C"
+                       parameter="loss_onoff" name="Loss On/Off" tooltip="Turns the loss filters on or off."/>
+        </View>
+        <View tab-caption="Degrade" padding="0" flex-direction="column" background-color="FF31323A"
+              margin="0">
+          <View flex-direction="column" max-height="140" background-color="FF1E1F22"
+                flex-grow="2.0" flex-shrink="2.0" padding="0">
+            <View max-height="5"/>
+            <Slider parameter="deg_depth" caption="Depth" class="Slider" name="Depth"
+                    tooltip="Sets the depth of the tape degradation." slider-type="linear-horizontal"
+                    max-height="70" margin="0" min-height="0" slidertext-width="80"
+                    flex-grow="2.5" flex-shrink="2.5" padding=""/>
+            <TextButton flex-align-self="auto" max-height="160" background-color="FF31323A"
+                        button-on-color="FFB41717" button-color="FF33343D" text="0.1x"
+                        button-off-text="FFFFFFFF" button-on-text="FFFFFFFF" height=""
+                        width="" min-height="0.0" padding="0" caption-placement="centred"
+                        min-width="" margin="10" caption="" flex-grow="1.0" flex-shrink="0.1"
+                        name="point1x" lookAndFeel="LookAndFeel_V3" tooltip="Scales the Depth value by 0.1 to allow for more subtle degradation"/>

```

Comment:
I'm not sure about the button name being "point1x", since the name is displayed for the tooltip as well. Maybe even "0.1x" would be better.